os/fs/smartfs/smartfs_smart.c

```diff
@@ -145,6 +145,7 @@ const struct mountpt_operations smartfs_operations = {
 	smartfs_sync,				/* sync */
 	smartfs_dup,				/* dup */
 	smartfs_fstat,				/* fstat */
+	NULL,
```

Comment:
Mention funcptr variable in comment section as reference.
Like /* truncate */

Reply:
Done